components/organisms/ToolList/nav-item.tsx

```diff
@@ -39,18 +39,9 @@ const NavItem: React.FC<NavItemProps> = ({ username, filterName, tool, selectedF
           selectedTool === tool.name.toLowerCase() ? "after:bg-orange-500" : "focus:after:bg-slate-400"
         } focus:bg-slate-100 focus:ring-slate-300 child:flex child:items-center`}
       >
-        <span
-          className={
-            "text-base whitespace-nowrap " +
-            (selectedTool === tool.name.toLowerCase() ? "text-slate-900" : "text-slate-500")
```

Comment:
`text-slate-500` was making things inacessbile from a colour contrast point of view.